modules/processing/snowflake/doc/_INTRO.md

```diff
@@ -0,0 +1,5 @@
+## processing
+
+<div class="badge core"></div>
```

Comment:
Before submitting please replace this for:
<div class="badges"><div class="core"></div></div>